serverless.yml

```diff
@@ -0,0 +1,59 @@
+# This file is the main config file for your service.
+
+service: dmk-test-service
+custom:
+  service-name: dmk-test-service
+
+provider:
```

Comment:
Please add the bucketName to the proivder properties so it doesn't go making a bunch of s3 buckets on the accounts I think it is serverless-deploy-{env} but I would double check in s3 before putting anything down

Reply:
Good idea. I just changed it to use the `serverless-us-west-1-{stage}-deploys` set of buckets.